helper_functions/ui2dash.m

```diff
@@ -91,5 +92,34 @@
                 'id', id, ...
                 'value', values, ...
                 'style', py.dict(pyargs('width', '50%', 'height', 150, 'font-size', 14))));
+
+            
+        % Label Properties
+        case 'uilabel'
+            lbl = ui_widget;
+            component = py.dash_html_components.Label(pyargs(...
+                'id',id, 'children', lbl.Text));
+
+        % Hyperlink Properties
+        case 'uihyperlink'
+            hlink = ui_widget;
+            component = py.dash_html_components.Img(pyargs(...
```

Comment:
maybe it should be py.dash_html_components.Link

Reply:
> maybe it should be py.dash_html_components.Link

Yes, fixed!